components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.spec.ts

```diff
@@ -24,4 +23,62 @@ describe('ProjectsDropdownComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('#projectsArray', () => {
+    it('#projectsArray returns an object naturally ordered by name', () => {
```

Comment:
... Even if you want to keep it on L27, an octothorp is not used with `it` anywhere else in the code base; but that is moot; should not be repeating the phrase from the enclosing `describe`:
```suggestion
    it('returns an object naturally ordered by name', () => {
```
Otherwise, in the test output you would see:
```
projectsArray projectsArray returns an object naturally ordered by name
```

Reply:
good point - updated 👍